### ISSUE TITLE
Rename mLayer to mTextureTarget

### ISSUE THREAD

**Issue by Michael-Lfx** [Think-Silicon/GLOVE#49]
By watching usage of accessor method of `mLayer`, it seems that renaming `mLayer` to `mTextureTarget` results in a better match.

https://github.com/Think-Silicon/GLOVE/blob/54d98bfa79383cdc2714e706555dd11bfb70e604/GLES/source/resources/attachment.h#L35

**Comment by abasilak** [comment]
Yes it can. However, a lot of functions (Get/Set) have to be renamed accordingly.

**Comment by Michael-Lfx** [comment]
What about I commit a PR to do it?

**Comment by Michael-Lfx** [comment]
Thank you.
